docs/src/app/components/pages/components/List/ExampleNested.js

```diff
@@ -6,37 +6,76 @@ import ContentInbox from 'material-ui/svg-icons/content/inbox';
 import ContentDrafts from 'material-ui/svg-icons/content/drafts';
 import ContentSend from 'material-ui/svg-icons/content/send';
 import Subheader from 'material-ui/Subheader';
+import Toggle from 'material-ui/Toggle';
```

Comment:
I have justed checked out the PR on my local env. I think that we can keep the example like this 👍 .
We already have a lot of them.